india_forecast_app/models/pvnet/utils.py

```diff
@@ -115,10 +128,7 @@ def process_and_cache_nwp(source_nwp_path: str, dest_nwp_path: str):
         if ds[v].dtype == object:
             ds[v].encoding.clear()
 
-    is_gfs = "gfs" in source_nwp_path.lower()
-    is_ecmwf = "ecmwf" in source_nwp_path.lower()
-
-    if is_ecmwf:
+    if nwp_config.source == "ecmwf":
```

Comment:
Might be a separate PR but it would be good to do this for ECMWF as well as there is also a difference in spatial resolution between training and live for it, although it is worth noting that I think we the current HF config files we use it will error since we have hacked them to work and would need to change them back to the original number of pixels

Reply:
yep, i was thinking a seperate PR for ECMWF stuff